### PR TITLE
native: update detail views to match figma designs

### PR DESCRIPTION
**Pull request by patosullivan**
This PR updates the notebook and gallery detail views to match the figma designs. It also updates the MessageInput to match figma throughout the app (smaller, less padding).

~~WIP, still needs:~~

- [x] ~~buttons to navigate between gallery posts while viewing a post~~ knowing which post comes next or after the current post, in this context, is fairly involved. Should be another issue/PR later imo.
- [x] ~~some idea of what we should do with the bottom bar when we have no content (looks funky, see attached video).~~ fixed

Latest state of things (updated since draft PR):

https://github.com/tloncorp/tlon-apps/assets/1221094/868645c4-1792-4cf8-a86d-bddf4759b266

### CODE REVIEW

**Comment by patosullivan** [comment]
I need to remove the blur from the bottom nav/message input. Should I move the back button back to the header and move the title back to the left within the header? cc @jamesacklin @urcades

**Comment by jamesacklin** [comment]
Eliminate blur, move < button back into the header. More to come in Episode 2.

**Comment by patosullivan** [comment]
Ok, eliminated blur and moved the back button back to the header.
Simulator Screenshot - iPhone 15 Pro - 2024-06-06 at 11 49 47

**Comment by patosullivan** [comment]
Noticing an issue with the smaller input having scroll bars on android, looking into it.

**Comment by jamesacklin** [comment]
I know we made the input taller to counter for the space between the top of the "Reply to post" text and the top edge of the input (misaligned with the bottom)—is there any way to make that space smaller and vertically center the text?

**Review by jamesacklin — APPROVED** [review]
Looks great. Filing a couple follow-on spacing/formatting issues, but the views themselves act as expected. Thank you!

File: packages/editor/src/MessageInputEditor.tsx [hunk 1]
```diff
@@ -75,8 +75,9 @@ export const MessageInputEditor = () => {
       style={{
         overflow: 'auto',
         height: 'auto',
-        // making this explicit
+        minHeight: 32,
```
Comment (dnbrwstr) [inline packages/editor/src/MessageInputEditor.tsx:78]: Is this value used anywhere else / referred to anywhere else / anything we can comment up?

File: packages/ui/src/components/GalleryPost/GalleryPost.tsx [hunk 2]
```diff
@@ -102,77 +96,46 @@ export default function GalleryPost({
...
-            height={detailView ? HEIGHT_DETAIL_VIEW_CONTENT : HEIGHT_AND_WIDTH}
+        {(postIsJustImage || textPostIsJustLinkedImage) && (
+          <ImageBackground
+            source={{ uri: postIsJustImage ? image!.src : linkedImage }}
+            style={{
+              width: HEIGHT_AND_WIDTH,
+              height: HEIGHT_AND_WIDTH,
+            }}
+            imageStyle={{
+              borderRadius: 12,
```
Comment (dnbrwstr) [inline packages/ui/src/components/GalleryPost/GalleryPost.tsx:107]: Can we use a theme value here?

File: packages/ui/src/components/GalleryPost/GalleryPost.tsx [hunk 3]
```diff
@@ -68,25 +55,32 @@ export default function GalleryPost({
     [post]
   );
 
+  const textPostIsJustReference = useMemo(
```
Comment (dnbrwstr) [inline packages/ui/src/components/GalleryPost/GalleryPost.tsx:58]: Can we extract these helpers + combine with the ones in `GalleryDetailView`? Could be contained in a `getPostMeta` hook or something like that.

File: packages/ui/src/components/MessageInput/index.native.tsx [hunk 4]
```diff
@@ -85,7 +90,7 @@ const getInjectedJS = (bridgeExtensions: BridgeExtension[]) => {
 
 // 52 accounts for the 16px padding around the text within the input
 // and the 20px line height of the text. 16 + 20 + 16 = 52
-const DEFAULT_CONTAINER_HEIGHT = 52;
+export const DEFAULT_MESSAGE_INPUT_HEIGHT = 44;
```
Comment (dnbrwstr) [inline packages/ui/src/components/MessageInput/index.native.tsx:93]: Update comment

File: packages/ui/src/components/MessageInput/index.native.tsx [hunk 5]
```diff
@@ -349,7 +355,6 @@ export const MessageInput = forwardRef<MessageInputHandle, MessageInputProps>(
 
               // @ts-expect-error setContent does accept JSONContent
               editor.setContent(newJson);
-              // editor.setSelection(initialSelection.from, initialSelection.to);
```
Comment (dnbrwstr) [inline packages/ui/src/components/MessageInput/index.native.tsx:352]: Intentional?

**Review by dnbrwstr — APPROVED** [review]
Looking good! Mostly minor nits.

File: packages/ui/src/components/GalleryPost/GalleryPost.tsx (hunk 2 above)
Comment (dnbrwstr) [reply packages/ui/src/components/GalleryPost/GalleryPost.tsx:107]: You may want to use `getTokenValue`

File: packages/ui/src/components/MessageInput/index.native.tsx (hunk 5 above)
Comment (patosullivan) [reply packages/ui/src/components/MessageInput/index.native.tsx:352]: Was old/commented out code that we didn't need. Was leftover from something I had tried a while ago.

**Pull request merged**